server/src/test/java/io/crate/lucene/NestedArrayLuceneQueryBuilderTest.java

```diff
@@ -132,4 +133,50 @@ public void test_any_not_equals_on_array_literal_and_nested_array_ref_with_autom
         var query = convert("[1] != any(c)");
         assertThat(query.toString()).isEqualTo("([1] <> ANY(array_unnest(c)))");
     }
+
+    @Test
+    public void test_lengths_of_nested_array_of_objects() throws Exception {
+        QueryTester.Builder builder = new QueryTester.Builder(
+            THREAD_POOL,
+            clusterService,
+            Version.CURRENT,
```

Comment:
Just double checked, updating this to `V_5_9_0` still passes.

Reply:
If I create the table on `5.9` and execute the queries on `5.10`, there are failures. I will create a follow up PR.